java/INSTALL

```diff
@@ -103,18 +108,25 @@ general overview of OpenDDS.
      use paths with no spaces (true on Unix too but spaces are rarely seen).
      The top level of the OpenDDS distribution is a directory that contains
      the subdirectory named dds.
+     
+     
+  3. Find the full path to JDK
+  
+     Linix    => export JAVA_HOME=$(readlink -f /usr/bin/javac | sed "s:/bin/javac::")
```

Comment:
OK, I’ll work on it based on your proposal.


> On Sep 9, 2016, at 1:50 PM, Adam Mitz <notifications@github.com> wrote:
> 
> In java/INSTALL <https://github.com/objectcomputing/OpenDDS/pull/300#discussion_r78230492>:
> 
> > @@ -103,18 +108,25 @@ general overview of OpenDDS.
> >       use paths with no spaces (true on Unix too but spaces are rarely seen).
> >       The top level of the OpenDDS distribution is a directory that contains
> >       the subdirectory named dds.
> > +     
> > +     
> > +  3. Find the full path to JDK
> > +  
> > +     Linix    => export JAVA_HOME=$(readlink -f /usr/bin/javac | sed "s:/bin/javac::")
> If the user has supplied "--java" on the command line (with no =), I think there are a few cases we can handle (regardless of platform):
> 
> User has set JAVA_HOME in the environment -> use it
> javac is found on PATH -> use location of bin/javac (after resolving symlinks on Unix) as JAVA_HOME
> Error -> message about supplying full path to the JDK installation or setting up PATH to include javac
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/objectcomputing/OpenDDS/pull/300/files/094dcd95acb650c544f5c0a592f64d36ff90bfb1#r78230492>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ADwEI5k5Jtd6qpHDNxwCZi_Dv7902Agjks5qoapxgaJpZM4J5UPs>.
>